src/core/a-cubemap.js

```diff
@@ -1,5 +1,6 @@
 /* global customElements, HTMLElement */
 var debug = require('../utils/debug');
+var ANode = require('./a-node').ANode;
```

Comment:
Not thrilled about adding an import here - but it seemed better to keep the logic consistent